### PR TITLE
Enable ios push notifications

### PR DESCRIPTION
**Pull request by cammellos**
Enable IOS push notification.

### Testing

I have tested on new accounts, and it should work. I haven't tested on existing account, but it should also be working, though not 100% sure. If it doesn't on existing account, I would address separately.

### CODE REVIEW

**Review by qfrank — APPROVED** [review]

**Review by smohamedjavid — APPROVED** [review]

**Review by ibrkhalil — APPROVED** [review]

**Review by erikseppanen — APPROVED** [review]

File: status-go-version.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.151.5",
-    "commit-sha1": "31d02f5197684317ad9f14874b0980d5ca716be6",
-    "src-sha256": "0ih77pq3lbk6qadv018pp7gzd6vhgzcl34vm1vg2mdmv2f15zvlw"
+    "version": "ios/push-notifications",
```
Comment (erikseppanen) [inline status-go-version.json]: Is this ok? It doesn't need to be v0.151.6?

Comment (cammellos) [reply status-go-version.json]: until it's tested and ready to merge, we set it to branch, we create a tag/version only after it's merged on status-go, then we update here before merging

**Comment by siddarthkay** [comment]
related : https://github.com/status-im/status-mobile/issues/14704

**Comment by pavloburykh** [comment]
Hi @cammellos! Thnx for the PR. Could you please rebase it to the latest develop before we proceed with testing? Thnx!

**Comment by pavloburykh** [comment]
> Hi @cammellos! Thnx for the PR. Could you please rebase it to the latest develop before we proceed with testing? Thnx!

Oh, I see it has been already rebases 3 days ago. Thank you, we can proceed with testing

**Comment by cammellos** [comment]
@pavloburykh the change has been merged already in develop (it was just a change on fleet),  would you mind testing there please?

**Comment by pavloburykh** [comment]
> @pavloburykh the change has been merged already in develop (it was just a change on fleet), would you mind testing there please?

sure!

**Pull request closed without merging**